### ISSUE TITLE
[Proposal]: Make the requirement to finish two Empire cargo missions more obvious

### ISSUE THREAD

**Issue by clintonmead** [naev/naev#2545]
### Suggestion

The requirement to do two empire cargo missions before continuing the campaign is somewhat non-obvious and can be forgot about. 

So perhaps do either of the two following things:

1. Make the requirement to do two empire cargo missions a mission itself, so you have such a mission in your log to complete OR
2. Don't make the requirement to do two cargo missions stop the spaceport bar mission from appearing, but instead makes it so he does appear and just reminds you of the requirement to go do two empire cargo missions.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bobbens** [comment]
Yes, I think having the NPC appear and basically say the requirements would be the best approach.

Czesc could tell the player to try to do missions (and/or requirements could be also reduced or eliminated).
Similarly the Dvaered-side could say that the player is not trust worthy enough or something along those lines. "Maybe if you help out House Dvaered more, I could give you a job" or something like that (better written of course).

**Comment by bobbens** [comment]
I've modified the long distance empire shipping missions and dvaered recruitment ones to inform of the requirements (2 ES missions or >= 20 standing). For things like negative standing, I don't think it's necessary. If there's any other mission that needs the same treatment I can look into it.
